specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1663 @@
+    "operationId": "IotDpsResource_ListBySubscription",
+    "x-ms-examples": {
+        "DPSListBySubscription": { "$ref": "./examples/DPSListBySubscription.json" }
+    },
+    "parameters": [
+     {
+      "$ref": "#/parameters/subscriptionId"
+     },
+     {
+      "$ref": "#/parameters/api-version"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "This is a synchronous operation. The body contains a JSON-serialized array of the metadata from all the provisioning services in the subscription.",
+      "schema": {
+       "$ref": "#/definitions/ProvisioningServiceDescriptionListResult"
+      }
+     },
+     "default": {
+      "description": "DefaultErrorResponse",
+      "schema": {
+       "$ref": "#/definitions/ErrorDetails"
+      }
+     }
+    },
+    "x-ms-pageable": {
+     "nextLinkName": "nextLink"
+    }
+   }
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices": {
+   "get": {
+    "tags": [
+     "GET"
+    ],
+    "operationId": "IotDpsResource_ListByResourceGroup",
+    "x-ms-examples": {
+        "DPSListByResourceGroup": { "$ref": "./examples/DPSListByResourceGroup.json" }
+    },
+    "consumes": [],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "$ref": "#/parameters/subscriptionId"
+     },
+     {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/api-version"
+     }
+    ],
+    "responses": {
+     "200": {
+      "description": "OK",
+      "schema": {
+       "$ref": "#/definitions/ProvisioningServiceDescriptionListResult"
+      }
+     },
+     "default": {
+      "description": "DefaultErrorResponse",
+      "schema": {
+       "$ref": "#/definitions/ErrorDetails"
+      }
+     }
+    },
+    "x-ms-pageable": {
+     "nextLinkName": "nextLink"
+    }
+   }
+  },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}/operationresults/{operationId}": {
+   "get": {
+    "tags": [
+     "GET"
+    ],
+    "operationId": "IotDpsResource_GetOperationResult",
+    "x-ms-examples": {
+        "DPSGetOperationResult": { "$ref": "./examples/DPSGetOperationResult.json" }
+    },
+    "consumes": [],
+    "produces": [
+     "application/json"
+    ],
+    "parameters": [
+     {
+      "name": "operationId",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "$ref": "#/parameters/subscriptionId"
+     },
+     {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "provisioningServiceName",
+      "in": "path",
+      "required": true,
+      "type": "string"
+     },
+     {
+      "name": "asyncinfo",
+      "in": "query",
+      "description": "asyncinfo flag",
```

Comment:
Description does not indicate the purpose of this property.
If this is a boolean, this should have a `boolean` type